docs/deployment.md

```diff
@@ -77,6 +70,12 @@ or, if you don't have the source code
 docker run -it --rm jaegertracing/jaeger-collector /go/bin/collector-linux -h
 ```
 
+Example:
+
+```
+docker run -it --rm -p14267:14267 -p14268:14268 jaegertracing/jaeger-collector /go/bin/collector-linux --cassandra.keyspace=jaeger_v1_test --cassandra.servers=192.168.0.183
```

Comment:
It might be more readable if you break the lines, like this:

```
docker run -it \
  --rm \
  --name jaeger-collector \
  -p14267:14267 \
  -p14268:14268 \
  --link cassandra:cassandra \
  jaegertracing/jaeger-collector \
  /go/bin/collector-linux --cassandra.keyspace=jaeger_v1_local --cassandra.servers=cassandra
```

Or, using env vars:

```
docker run -it \
  --rm \
  --name jaeger-collector \
  -p14267:14267 \
  -p14268:14268 \
  -e CASSANDRA_KEYSPACE=jaeger_v1_local \
  -e CASSANDRA_SERVERS=cassandra \
  --link cassandra:cassandra \
  jaegertracing/jaeger-collector
```

(note that I changed the keyspace to be consistent with the previous command that starts Cassandra, and added a link to that container, so that we don't need IPs that won't work for the reader)